modules/react/modal/stories/Modal.stories.mdx

```diff
@@ -81,6 +83,31 @@ requires a `label` prop.
 
 <ExampleCodeBlock code={CustomTarget} />
 
+### Body Content Overflow
+
+There are a few ways to handle body content overflowing. The easiest is to use `Modal.Card` as a
+vertical stack using `VStack` and setting the max height to the view height minus the desired margin
+around the card (default margin is `xl` or `40px`). The `VStack` is a flexbox container than
+controls the height of the children. From there, you set up the content area to have an
+`overflow: scroll` style. Usually, you want the header and action buttons to stay put. In this
+example, the `Modal.Card` has an `as={VStack}` with no spacing, and a max height of
+`calc(100vh - ${space.xl} * 2)` which is the viewport height minus the default margin x 2 (top and
+bottom). The `Modal.Heading`, `Modal.Body`, and `HStack` are the 3 flexbox children that will try to
+equally fill the container up to the max height defined. The `overflow: scroll` on the `Modal.Body`
+is what restricts the content to an overflow container so that it scrolls. The effectively fixes the
+position of the heading and action buttons.
+
```

Comment:
This i an awesome description, i would just break it up into smaller paragraph or bullet points so it's a little easier to follow